packages/commercetools/theme/nuxt.config.js

```diff
@@ -32,10 +32,7 @@ export default {
       },
       {
         rel: 'stylesheet',
-        href: 'https://fonts.googleapis.com/css?family=Raleway:300,400,400i,500,600,700|Roboto:300,300i,400,400i,500,700&display=swap',
-        media: 'print',
-        onload: 'this.media=\'all\'',
-        once: true
+        href: 'https://fonts.googleapis.com/css?family=Raleway:300,400,400i,500,600,700|Roboto:300,300i,400,400i,500,700&display=swap'
```

Comment:
Can we keep `media` and `onload`, but only remove `once`?

Reply:
@filipsobol I've already tried that, but I encountered strange behaviour with constant re-generating font `<link>` in page header on every reactive event 
![image](https://user-images.githubusercontent.com/32792205/127651811-3147dfe7-31f0-41b5-9ca2-79354333e21f.png)

and also with disabled cache in browser (important) font is still fetched multiple times
![image](https://user-images.githubusercontent.com/32792205/127651525-ec3b2a46-1681-462b-8ca6-e388d390737d.png)